### PR TITLE
Project 1: Setup and mobile version skeleton

### PR DESCRIPTION
**Pull request by Kambuchi**
## Mobile version hero

- Basic files created
- Hero: content and style added
- Images: icons & background added

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  margin-left: 24px;
+  margin-right: 35px;
+  font-weight: 700;
+  color: #6070ff;
+  font-size: 18px;
+  padding-bottom: 10px;
+}
```
Comment (safafa) [inline style.css:26]: - [ ] To better follow the DRY concept you can refactor the code by grouping repeated CSS code into a separate class.  for example : 
`.flex {
display: flex;
}`

- [ ] [OPTIONAL] Check this [link](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_propertiesl) about setting CSS variables if you want to make your code better.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+            <p>I’m a software developer! I can help you build a product, 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <span>LET'S CONNECT</span>
+            <ul class="hero-icons">
+                <li><img src="./img/ic_twitter.svg" alt="twitter profile"></li>
+                <li><img src="./img/ic_linkedin.svg" alt="linkedin profile"></li>
+                <li><img src="./img/ic_medium.svg" alt="medium profile"></li>
+                <li><img src="./img/ic_github.svg" alt="github profile"></li>
+                <li><img src="./img/ic_angellist.svg" alt="angellist profile"></li>
```
Comment (safafa) [inline index.html:30]: - [ ] All the icons are clickable. Kindly use the correct HTML tag. As a suggestion, you can use a or button tag.

**Review by safafa — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED ♻️ 

Hi @Kambuchi , You did a great job so far 👏🏼 👏🏼 

- The repository looks professional.
- You did well in implementing the project specifications and following the Figma design. 

However, here are some points that you need to work on to get your project approved.

## Required changes :

- [ ] Check the inline comments.

- [ ]  Kindly check this [link](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/) about how to write meaningful commit messages

## Optional changes :
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Kambuchi Pereira Montañez, 
## Congratulations 🎉 Thank you for implementing changes suggested previously 🙏

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**